packages/react/emotion-utils/src/flex.tsx

```diff
@@ -7,27 +7,31 @@ export interface FlexOptions {
   align?: CSSProperties['alignItems'];
   justify?: CSSProperties['justifyContent'];
   direction?: CSSProperties['flexDirection'];
+  gap?: CSSProperties['gap'];
 }
 
 export function flex(options: FlexOptions): SerializedStyles;
 export function flex(
   align: CSSProperties['alignItems'],
   justify?: CSSProperties['justifyContent'],
-  direction?: CSSProperties['flexDirection']
+  direction?: CSSProperties['flexDirection'],
+  gap?: CSSProperties['gap']
 ): SerializedStyles;
 export function flex(
   alignOrFlexOptions: FlexOptions | CSSProperties['alignItems'],
   justify = 'flex-start',
-  direction = 'row'
+  direction = 'row',
+  gap = 0
```

Comment:
`CSSProperties['gap']` can be  `Property.Gap<string | number> | undefined`

Is it okay to fix the default to zero?

Reply:
The default of gap is that it doesn't need gap, so I don't think it'll be a problem even if it's 0

Reply:
I think this could be type error for overriding

<img width="788" alt="image" src="https://github.com/toss/slash/assets/57122180/e16be2b5-4ca4-4094-a986-8bac91fa5324">